src/diffpy/labpdfproc/tools.py

```diff
@@ -174,16 +176,33 @@ def _load_wavelength_from_config_file(args):
     args : argparse.Namespace
         The updated arguments with the updated wavelength and anode type.
     """
-    if args.wavelength or args.anode_type:
-        return args
     global_config = _load_config(Path().home() / "diffpyconfig.json")
     local_config = _load_config(Path().cwd() / "diffpyconfig.json")
-    if local_config:
-        args.wavelength = local_config.get("wavelength")
-        args.anode_type = local_config.get("anode_type")
-    elif global_config:
-        args.wavelength = global_config.get("wavelength")
-        args.anode_type = global_config.get("anode_type")
+    local_has_data = local_config and (
+        "wavelength" in local_config or "anode_type" in local_config
+    )
+    global_has_data = global_config and (
+        "wavelength" in global_config or "anode_type" in global_config
+    )
+    if not local_has_data and not global_has_data:
+        print(
+            "No configuration file was found containing information "
+            "about the wavelength or anode type. \n"
+            "You can add the wavelength or anode type "
+            "to a configuration file on the current computer "
+            "and it will be automatically associated with "
+            "subsequent diffpy data by default. \n"
+            "You will only have to do that once. \n"
+            "For more information, please refer to www.diffpy.org/"
+            "diffpy.labpdfproc/examples/toolsexample.html"
+        )
+
+    if args.wavelength or args.anode_type:
+        return args
+    config = local_config if local_has_data else global_config
+    if config:
+        args.wavelength = args.wavelength or config.get("wavelength")
+        args.anode_type = args.anode_type or config.get("anode_type")
```

Comment:
this is all very nicely done.  I like the coding style very much.  Very readable.